### ISSUE TITLE
INFO: Reproducible Builds IRC Meeting

### ISSUE THREAD

**Issue by josephbisch** [devrandom/gitian-builder#136]
I tried emailing the following information to @devrandom, but apparently the email is no longer monitored, so I am filing it as an issue here. Sorry for filing an issue that isn't actually an issue with gitian-builder.

There are going to be regular IRC meetings about reproducible builds in general organized by the reproducible-builds.org folks.

Here is the information on the next one (there already was one):
http://lists.reproducible-builds.org/pipermail/rb-general/2016-October/000071.html

Next one is Monday October 17. I'm going to try to be there for the meeting. Maybe I'll mention Gitian-related recent developments if you aren't there to mention them yourself. Let me know if there is anything you would like me to bring up.

**Comment by devrandom** [comment]
Thank you for the heads-up, I will attend.

**Comment by devrandom** [comment]
Sent you a message on IRC.  Please let me know here if you still have an issue reaching me.

**Comment by josephbisch** [comment]
Thanks. I don't normally need to contact you, so it isn't an issue, but when I tried emailing nine days ago, I got the following back from that same email address:

```
This mailbox is not monitored.  If you need to reach me, please open a ticket in one of my projects, or find another channel.  Sorry for the inconvenience, but this is a spam trap.
```

I'm sure it is the same address you just sent. Just FYI, in case you have some setting on that you didn't intend to have on telling people the mailbox isn't monitored when it really is.
